components/usage/pkg/db/workspace_instance.go

```diff
@@ -79,6 +79,7 @@ func ListWorkspaceInstancesInRange(ctx context.Context, conn *gorm.DB, from, to
 		).
 		Where("creationTime < ?", TimeToISO8601(to)).
 		Where("startedTime != ?", "").
+		Where("usageAttributionId != ?", "").
```

Comment:
How will we want to handle potentially un-attributed instances?

Reply:
Given the `usageAttributionId` should always be present (the premise of your PR). The check here mostly ensures we only run on "new" data which does have the attribution set